docs/intro.rst

```diff
@@ -314,17 +314,17 @@ This is the current list of error and warning codes:
 +------------+----------------------------------------------------------------------+
 | **E3**     | *Blank line*                                                         |
 +------------+----------------------------------------------------------------------+
-| E301       | expected 1 blank line, found 0                                       |
+| E301       | expected 1 blank line, found …                                       |
```

Comment:
This now exceeds the stated purpose of this pull request and will break downstream redistributors who don't have an internationalized locale on their builders and CI infrastructure. Please do not use unicode here.